docs/GettingStarted.md

```diff
@@ -2766,6 +2766,30 @@ When profiling [Resque](https://github.com/resque/resque) jobs, you should set t
 
 Without this flag, profiles for short-lived Resque jobs will not be available as Resque kills worker processes before they have a chance to submit this information.
 
+### Error Tracking
+
+`datadog` can automatically report handled errors. The errors are attached through span events to the span in which they are handled. They are also directly reported to Error Tracking.
+
+#### Requirements
+
+- Ruby 2.6+. JRuby and TruffleRuby are not supported.
+- datadog 2.16.0+.
+
+#### Configuration
+
+To enable automatic reporting of handled errors, one of the two environement variables must be set:
+- `DD_ERROR_TRACKING_HANDLED_ERRORS` = `user|third_party|all`. Report handled errors of user code, gems or both.
+- `DD_ERROR_TRACKING_HANDLED_ERRORS_INCLUDE` = `file, gems, path..`. A list of comma-separated paths, file names
+and gem names for which handled errors will be reported.  
```

Comment:
```suggestion
- `DD_ERROR_TRACKING_HANDLED_ERRORS_INCLUDE` = `file, gems, path..`. A list of comma-separated paths, file names,
and gem names for which handled errors will be reported.
For Ruby 3.3 or newer, the location where the error was raised and where it was rescued will be matched. For earlier versions of Ruby, only the location where the error was raised can be matched.
```

Reply:
addressed in 6d7518177cee5d42dd37335629e76cd5a381205e